.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,9 @@
+name: GitHub Actions - Check PR Title
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+jobs:
+  Jira-PR-Title:
+    if: ${{ (github.actor != 'dependabot[bot]') && (github.actor != 'EburyCrowdin') }}
```

Comment:
We don't have EburyCrowdin in chameleon

Reply:
That is fine. It is a general statement covering multiple actors within the Ebury organisation 👍🏾